tools/marvin/marvin/config/test_data.py

```diff
@@ -928,6 +928,16 @@
         "ostype": "CentOS 5.6 (64-bit)",
         "mode": 'HTTP_DOWNLOAD',
     },
+    "iso3": {
+        "displaytext": "Test ISO 3",
+        "name": "ISO 3",
+        "url": "http://people.apache.org/~tsp/dummy.iso",
```

Comment:
URL is dummy, is there a valid iso url we can use?

Reply:
@rhtyd, I could add a valid URL, but what is the reason? I've followed the rest of Marvin's test, and no one is using a valid ISO, only dummies

Reply:
@slavkap does your newly added test work with the fix?

Yes, that's a practice we should avoid if/when possible. For running tests with blueorangutan/Trillian, we patch the test_data.py file https://github.com/shapeblue/Trillian/blob/master/Ansible/roles/marvin/templates/test_data.py.j2

So anytime anyone adds invalid data, we manually patch it.

Reply:
@rhtyd, I tested it with UEFI enabled/disabled, and the test works as expected, but it only checks the dumpxml of the domain, nothing more (that's why I don't see a reason to add a valid ISO). Do I have to open a PR with the changes of test_data.py to your repo?

Reply:
Ah okay so the invalid ISO doesn't make any difference. No need since by default Trillian env don't have UEFI enabled hosts, the tests will be skipped. I'll merge the PR based on your remark.